services/plantService.js

```diff
@@ -0,0 +1,11 @@
+const plantDao = require("../models/plantDao");
+
+const getPlantInfo = async (plantId) => {
+  const PlantInfo = await plantDao.getPlantInfo(plantId);
+
+  return PlantInfo;
+};
```

Comment:
```suggestion
const getPlantInfo = async (plantId) => {
  return plantDao.getPlantInfo(plantId);
};
```
변수 선언 없이 바로 return할 수 있는 코드이니 수정해주세요!